### PR TITLE
chore(main-page): create layout

### LINKED ISSUE #15: Верстка главной страницы

### PR DESCRIPTION
**Pull request by MishaMgla**
init main page layout

issue #15

### CODE REVIEW

**Review by zena7 — APPROVED** [review]

File: src/app/ui/layout/main/main.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#main{
```
Comment (mg901) [inline src/app/ui/layout/main/main.css]: Replace id with css class

File: src/app/ui/layout/main/main.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    background-color: rgb(127, 29, 167);
+    width: 75%;
+    height: 1000px;
```
Comment (mg901) [inline src/app/ui/layout/main/main.css]: remove height

File: src/app/ui/layout/main/main.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import './main.css';
+
+function Main() {
+  return <main id="main"></main>;
```
Comment (mg901) [inline src/app/ui/layout/main/main.tsx]: replace id with className

File: src/app/app.tsx [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+import { Layout } from './ui/layout';
 
 import 'normalize.css';
 
 export function App() {
   return (
-    <Provider store={store}>
-      <Routes />
-    </Provider>
+    <Layout />
```
Comment (mg901) [inline src/app/app.tsx]: Remove Layout component

File: src/app/ui/layout/footer/footer.css [hunk 5]
```diff
@@ -0,0 +1,6 @@
+#footer{
```
Comment (mg901) [inline src/app/ui/layout/footer/footer.css]: replace id with css class

File: src/app/ui/layout/footer/footer.css [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
+    font-family: sans-serif;
+    width: 100%;
+    height: 40px;
```
Comment (mg901) [inline src/app/ui/layout/footer/footer.css]: remove height property

File: src/app/ui/layout/footer/footer.tsx [hunk 7]
```diff
@@ -0,0 +1,7 @@
+import './footer.css';
+
+function Footer() {
+  return <footer id="footer"></footer>;
```
Comment (mg901) [inline src/app/ui/layout/footer/footer.tsx]: replace id with className

File: src/app/ui/layout/header/header.css [hunk 8]
```diff
@@ -0,0 +1,6 @@
+#header {
```
Comment (mg901) [inline src/app/ui/layout/header/header.css]: replace id with css class

File: src/app/ui/layout/header/header.css [hunk 9]
```diff
@@ -0,0 +1,6 @@
...
+    width: 100%;
+    height: 50px;
+    position: fixed;
```
Comment (mg901) [inline src/app/ui/layout/header/header.css]: Why do you use position height?

File: src/app/ui/layout/header/header.tsx [hunk 10]
```diff
@@ -0,0 +1,7 @@
+import './header.css';
+
+function Header() {
+  return <header id="header"></header>;
```
Comment (mg901) [inline src/app/ui/layout/header/header.tsx]: replace id with className

File: src/app/ui/layout/index.tsx [hunk 11]
```diff
@@ -1,5 +1,18 @@
...
+import './layout.css';
+import Header from './header/header';
+import Main from './main/main';
+import Sidebar from './sidebar/sidebar';
+import Footer from './footer/footer';
 
 export const Layout = () => {
-  return <div>layout</div>;
+  return (
+    <div id="layout">
```
Comment (mg901) [inline src/app/ui/layout/index.tsx]: replace id with className

File: src/app/ui/layout/index.tsx [hunk 12]
```diff
@@ -1,5 +1,18 @@
...
+      <Header />
+      <div id="main-wrapper">
```
Comment (mg901) [inline src/app/ui/layout/index.tsx]: - Why do you use wrapper? 
- Replace id with className

File: src/app/ui/layout/layout.css [hunk 13]
```diff
@@ -0,0 +1,10 @@
+#layout {
```
Comment (mg901) [inline src/app/ui/layout/layout.css]: Replace id with css class

File: src/app/ui/layout/layout.css [hunk 14]
```diff
@@ -0,0 +1,10 @@
...
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+#main-wrapper {
```
Comment (mg901) [inline src/app/ui/layout/layout.css]: replace id with css class

File: src/app/ui/layout/sidebar/sidebar.css [hunk 15]
```diff
@@ -0,0 +1,4 @@
+#sidebar{
```
Comment (mg901) [inline src/app/ui/layout/sidebar/sidebar.css]: Replace id with css class

File: src/app/ui/layout/sidebar/sidebar.tsx [hunk 16]
```diff
@@ -0,0 +1,7 @@
+import './sidebar.css';
+
+function Sidebar() {
+  return <aside id="sidebar"></aside>;
```
Comment (mg901) [inline src/app/ui/layout/sidebar/sidebar.tsx]: Replace id with className

File: src/app/ui/footer/footer.tsx [hunk 17]
```diff
@@ -0,0 +1,7 @@
+import './footer.css';
+
+function Footer() {
+  return <footer className="footer"></footer>;
+}
+
+export default Footer;
```
Comment (mg901) [inline src/app/ui/footer/footer.tsx]: replace default export with named export

File: src/app/ui/header/header.tsx [hunk 18]
```diff
@@ -0,0 +1,7 @@
+import './header.css';
+
+function Header() {
+  return <div className="header"></div>;
+}
+
+export default Header;
```
Comment (mg901) [inline src/app/ui/header/header.tsx]: replace default export with named export

File: src/app/ui/main/main.tsx [hunk 19]
```diff
@@ -0,0 +1,7 @@
+import './main.css';
+
+function Main() {
+  return <main className="main"></main>;
+}
+
+export default Main;
```
Comment (mg901) [inline src/app/ui/main/main.tsx]: replace default export with named export

File: src/app/ui/sidebar/sidebar.tsx [hunk 20]
```diff
@@ -0,0 +1,7 @@
+import './sidebar.css';
+
+function Sidebar() {
+  return <aside className="sidebar"></aside>;
+}
+
+export default Sidebar;
```
Comment (mg901) [inline src/app/ui/sidebar/sidebar.tsx]: replace default export with named export

File: src/app/ui/footer/footer.tsx [hunk 21]
```diff
@@ -0,0 +1,5 @@
+import './footer.css';
+
```
Comment (mg901) [inline src/app/ui/footer/footer.tsx:2]: add type for Properties

File: src/app/ui/footer/footer.tsx [hunk 22]
```diff
@@ -0,0 +1,5 @@
...
+export function Footer() {
+  return <footer className="footer"></footer>;
```
Comment (mg901) [inline src/app/ui/footer/footer.tsx:4]: add children

File: src/app/ui/footer/footer.tsx [hunk 23]
```diff
@@ -0,0 +1,5 @@
+import './footer.css';
```
Comment (mg901) [inline src/app/ui/footer/footer.tsx]: replace `import './footer'.css` with `import 'footer.module.css`

File: src/app/ui/footer/footer.css [hunk 24]
```diff
@@ -0,0 +1,6 @@
+.footer{
```
Comment (mg901) [inline src/app/ui/footer/footer.css]: rename footer.css with footer.module.css

File: src/app/ui/header/header.tsx [hunk 25]
```diff
@@ -0,0 +1,5 @@
+import './header.css';
+
```
Comment (mg901) [inline src/app/ui/header/header.tsx:2]: add type for properties

File: src/app/ui/header/header.tsx [hunk 26]
```diff
@@ -0,0 +1,5 @@
...
+export function Header() {
+  return <div className="header"></div>;
```
Comment (mg901) [inline src/app/ui/header/header.tsx:4]: add children property

File: src/app/ui/layout/index.tsx [hunk 27]
```diff
@@ -1,5 +1,16 @@
-import './layout.scss';
+import './layout.css';
+import { Header } from '../header/header';
```
Comment (mg901) [inline src/app/ui/layout/index.tsx]: replace `'../header/header'; ` with `'../header';`

File: src/app/ui/layout/index.tsx [hunk 28]
```diff
@@ -1,5 +1,16 @@
...
+import { Main } from '../main/main';
```
Comment (mg901) [inline src/app/ui/layout/index.tsx]: replace `'../main/main'; ` with `'../main';`

File: src/app/ui/layout/index.tsx [hunk 29]
```diff
@@ -1,5 +1,16 @@
...
+import { Sidebar } from '../sidebar/sidebar';
```
Comment (mg901) [inline src/app/ui/layout/index.tsx]: replace `'../sidebar/sidebar'; ` with `'../sidebar';`

File: src/app/ui/layout/index.tsx [hunk 30]
```diff
@@ -1,5 +1,16 @@
...
+import { Footer } from '../footer/footer';
```
Comment (mg901) [inline src/app/ui/layout/index.tsx]: replace `'../footer/footer'; ` with `'../footer';`

File: src/app/ui/main/main.tsx [hunk 31]
```diff
@@ -0,0 +1,5 @@
+import './main.css';
+
```
Comment (mg901) [inline src/app/ui/main/main.tsx:2]: add type for properties

File: src/app/ui/main/main.tsx [hunk 32]
```diff
@@ -0,0 +1,5 @@
...
+export function Main() {
+  return <main className="main"></main>;
```
Comment (mg901) [inline src/app/ui/main/main.tsx:4]: add children property

File: src/app/ui/sidebar/sidebar.css [hunk 33]
```diff
@@ -0,0 +1,4 @@
+.sidebar {
```
Comment (mg901) [inline src/app/ui/sidebar/sidebar.css]: rename `sidebar.css` with `sidebar.module.css`

File: src/app/ui/sidebar/sidebar.tsx [hunk 34]
```diff
@@ -0,0 +1,5 @@
+import './sidebar.css';
```
Comment (mg901) [inline src/app/ui/sidebar/sidebar.tsx]: replace`./sidebar.css` with `./sidebar.module.css`

File: src/app/ui/main/main.tsx [hunk 35]
```diff
@@ -0,0 +1,5 @@
+import './main.css';
```
Comment (mg901) [inline src/app/ui/main/main.tsx]: replace`main.css` with `main.module.css`

File: src/app/ui/main/main.css [hunk 36]
```diff
@@ -0,0 +1,4 @@
+.main{
```
Comment (mg901) [inline src/app/ui/main/main.css]: rename `main.css` with `main.module.css`

**Pull request merged**